src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -984,6 +986,13 @@ public IEnumerable<IVsProjectAdapter> GetAllVsProjectAdapters()
             return _projectSystemCache.GetVsProjectAdapters();
         }
 
+        public async Task CleanAsync()
```

Comment:
Name of the method is ambiguous (clean what?). Also this has to be a utility method. Solution manager should stay just a data only class.